qrScanner/src/main/java/io/github/novacrypto/qrscanner/QrCamera.kt

```diff
@@ -68,12 +68,14 @@ internal class QrCamera(
             if (value == field) return
             field = value
             value?.let {
-                Snackbar.make(cameraSurfaceView, it, Snackbar.LENGTH_INDEFINITE)
-                        .setAction(android.R.string.ok, { _ ->
-                            Timber.d("Accepted barcode")
-                            onAccept(it)
-                        })
-                        .show()
+                if (!options.autoAcceptResult) {
+                    Snackbar.make(cameraSurfaceView, it, Snackbar.LENGTH_INDEFINITE)
+                            .setAction(android.R.string.ok, { _ ->
+                                onAccept(it)
+                            })
+                            .show()
+                } else onAccept(it)
+                Timber.d("Accepted barcode")
```

Comment:
This log is now in an incorrect place. Needs to show after `onAccept` and for the `Snackbar` path, it would actually show immediately now, instead of after.

Suggest: (note also inverting the `if` so it doesn't have a `!`)

```
            value?.let {
                if (options.autoAcceptResult) {
                    Timber.d("Accepted barcode automatically")
                    onAccept(it)
                 } else {
                    Snackbar.make(cameraSurfaceView, it, Snackbar.LENGTH_INDEFINITE)
                            .setAction(android.R.string.ok, { _ ->
                                Timber.d("Accepted barcode")
                                onAccept(it)
                            })
                            .show()
                }
            }
```